src/main/java/com/meetup/server/place/application/PlaceService.java

```diff
@@ -0,0 +1,35 @@
+package com.meetup.server.place.application;
+
+import com.meetup.server.event.domain.Event;
+import com.meetup.server.event.implement.EventReader;
+import com.meetup.server.place.domain.Place;
+import com.meetup.server.place.implement.PlaceReader;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.UUID;
+
+@Slf4j
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class PlaceService {
+
+    private final EventReader eventReader;
+    private final PlaceReader placeReader;
+
+    @Transactional
+    public void confirmPlace(UUID eventId, UUID placeId) {
+        Event event = eventReader.read(eventId);
+        Place place = placeReader.read(placeId);
+
+        if (event.getPlace() == null) {
```

Comment:
```suggestion
        if (event.hasNoPlace()) {
            confirm()
        } else {
            update()
```
도메인 메소드를 만들면 더욱 도메인에 응집된 코드가 될 수 있어요!